readme.txt

```diff
@@ -18,7 +18,7 @@ This is an open-source modern boilerplate / starter theme with all the latest de
 
 It is intended for users with at least some dev experience (php and js) and it's a great starting point if you want to build a theme for your own or client's project, to release a theme on WordPress.org or to sell it.
```

Comment:
Change the `It` to `Eightshift WordPress Boilerplate` for more clarity 🙂

Reply:
👍